src/ui/js/Game.js

```diff
@@ -1680,53 +1680,102 @@ Game.pageAddGameNavigationFooter = function() {
 
 // Display a footer-style message with the list of skills in this game
 Game.pageAddSkillListFooter = function() {
-  var gameSkillDiv = $('<div>', {
+  var hasButtonSpecial = false;
+  var hasDieSkill = false;
+  var isFirstButtonSpecial = true;
+  var isFirstDieSkill = true;
+  var isFirstInteraction;
```

Comment:
I kind of have a pet peeve about this isFirstWhatever logic style --- it seems to me that having an "if info.interacts.length > 0" block before the loop accomplishes what you want here, and is much tidier.